lib/algo/DMD.h

```diff
@@ -65,6 +65,11 @@ class DMD
      */
     DMD(std::string base_file_name);
 
+    /**
+     * @brief Set the offset of a certain order.
+     */
+    virtual void setOffset(Vector* state_offset, int order); 
```

Comment:
Although it does not affect results, would it be more accurate to use the name `offset_vector` instead of `state_offset`, since it could be a state or derivative offset? Also in `NonuniformDMD`.

Reply:
Thanks for spotting that, Dylan. I forgot to change the variable name in the header files.